zerver/lib/html_diff.py

```diff
@@ -85,9 +100,15 @@ def highlight_html_differences(s1, s2):
     idx = 0
     while idx < len(ops):
         op, text = ops[idx]
-        next_op = None
-        if idx != len(ops) - 1:
-            next_op, next_text = ops[idx + 1]
```

Comment:
What was the purpose of these lines here, earlier?
I couldn't see any direct use though, but still thought worth asking if you stumbled upon something relating to it.